base/base-guide.md

```diff
@@ -0,0 +1,151 @@
+# {Doctype} guide
+
+This {doctype}-guide provides extra writing tips for each of the sections within the {doctype}-template.
+
+**Version:** {MAJOR.MINOR.PATCH}
+
+**Last updated:** {Month DD, YYYY}
+
+## Prerequisites
+
+To make the best use of this guide, it helps if you have a working knowledge of:
+
+* The project's documentation style guide.
+* Technical writing basics, such as taught in Google's [technical writer training courses](https://developers.google.com/tech-writing).
+
+## {Doctype} content
+### Hero description
+The hero description stands alone, just under the title. It is sometimes referred to as the "TL;DR statement", short for: Too Long; Dont Read" and is written as:
+```
+   TL;DR: {Hero description.}
+```
+
+It should be very short so that a reader will naturally want to read it. It's purpose is to quickly help the reader decide whether they should read any further.
```

Comment:
**Grammar nitpick (non-blocking):** Should be: "*Its* purpose is to quickly..."

Reply:
Ack. Thanks.